src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -393,27 +452,47 @@ def __get_pydantic_json_schema__(
         return json_schema
 
 
-class ClassMeta(BaseModel):
+class MetadataBase(BaseModel):
+    """Base model for all root metadata models generated."""
+
     class_: enums.FMUClassEnum = Field(
         alias="class",
-        title="Metadata class",
+        title="metadata_class",
     )
+
     masterdata: Masterdata
+    """The ``masterdata`` block contains information related to masterdata.
+    See :class:`Masterdata`."""
+
     tracklog: List[TracklogEvent]
-    source: Literal["fmu"] = Field(description="Data source (FMU)")
-    version: Literal["0.8.0"] = Field(title="FMU results metadata version")
+    """The ``tracklog`` block contains a record of events recorded on these data.
+    See :class:`TracklogEvent`."""
+
+    source: Literal["fmu"]
+    """The source of this data. Defaults to 'fmu'."""
+
+    version: Literal["0.8.0"]
+    """The version of the schema that generated this data."""
 
 
-class FMUCaseClassMeta(ClassMeta):
+class CaseMetadata(MetadataBase):
+    """The FMU metadata model for an FMU case."""
```

Comment:
Is there a better name for this?

Reply:
Not that I can think of

Reply:
The "case" concept is tricky. It is highly conventional, but over the years I guess it has become so institutionalized that it is difficult to get away from. ERT however has no concept of the "case". There is the concept of "experiment", but these are not 1:1. It is also a bit chaotic with many different implementation of a corresponding variable across different FMU workflows. The _word_ "case" is also used to describe different things. E.g. some would refer to realizations as "cases" and so on. So there is some confusion.

We could consider adding a description here of what we mean with the term "case" in this context, perhaps in a note or something. This is the definition we have included in the Sumo docs:

```
A case represent a set of iterations that belong together, either by being part of the same run (i.e. history matching) or by being placed together by the user, corresponding to /scratch/<asset>/<user>/<my case name>/.
```

Reply:
Updated with this comment both here and on the `Case` model specifically